src/tokenizer.rs

```diff
@@ -95,11 +95,17 @@ impl<'o> Tokenize<'o, Vec<u8>> for &'o str {
 /// See [`TokenizerBuilder`] to know how to build a [`Tokenizer`].
 pub struct Tokenizer<'sw, A> {
     stop_words: Option<&'sw Set<A>>,
+    create_char_map: bool,
 }
 
 impl<'o, A: AsRef<[u8]>> Tokenizer<'_, A> {
     pub fn tokenize(&self, original: &'o str) -> ClassifiedTokenIter<'o, '_, A> {
-        original.segment().normalize().classify_with_stop_words(self.stop_words)
+        if self.create_char_map {
+            original.segment().normalize().classify_with_stop_words(self.stop_words)
+        } else {
+            let options = NormalizerOption { create_char_map: false };
+            original.segment().normalize_with_option(options).classify_with_stop_words(self.stop_words)
+        }
```

Comment:
You should always call `normalize_with_option`, the behavior would be the same but the code would be factorized.